CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,145 @@
+[![SnapIT](./src/assets/images/SnapIT-logo.png)](https://www.snapit.solutions)
+
+# SnapIT HacktoberFest Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socioeconomic status,
+nationality, personal appearance, race, caste, color, religion, or sexual identity
+and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+- Demonstrating empathy and kindness toward other people
+- Being respectful of differing opinions, viewpoints, and experiences
+- Giving and gracefully accepting constructive feedback
+- Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+- Focusing on what is best not just for us as individuals, but for the
+  overall community
+
+Examples of unacceptable behavior include:
+
+- The use of sexualized language or imagery, and sexual attention or
+  advances of any kind
+- Trolling, insulting or derogatory comments, and personal or political attacks
+- Public or private harassment
+- Publishing others' private information, such as a physical or email
+  address, without their explicit permission
+- Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Enforcement Responsibilities
+
+Community leaders are responsible for clarifying and enforcing our standards of
+acceptable behavior and will take appropriate and fair corrective action in
+response to any behavior that they deem inappropriate, threatening, offensive,
+or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
+decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at [hr@snapit.solutions](hr@snapit.solutions).
```

Comment:
This email address should not be a generic corporate email address, but an address that project maintainers have direct access to. For example, a distribution list that project maintainers are subscribed to, or their individual email addresses, as seen in the example of the git Code of Conduct page: https://github.com/git/git/blob/master/CODE_OF_CONDUCT.md

Reply:
Use hacktoberfest@snapit.solutions as the email address.

Reply:
Addressed: 903f3d616a442b096e919dbd947c6a5e4b63dffc

Reply:
Please adjust the displayed address. The page shows "hr@snapit.solutions" but links to "hacktoberfest@snapit.solutions".

Reply:
Addressed: ce5464932759df93a9e79a95563d67c7267b5de1